src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -688,6 +706,15 @@ internal Collection<FunctionDescriptor> GetFunctionDescriptors(IEnumerable<Funct
             return functionDescriptors;
         }
 
+        internal static bool FunctionsSupported(IEnumerable<FunctionMetadata> functions)
+        {
+            if (functions != null && functions.Any())
+            {
+                return functions.Where(funcMetadata => (funcMetadata.ScriptType != functions.ElementAt(0).ScriptType) || (funcMetadata.Language != functions.ElementAt(0).Language)).Any() == false;
+            }
+            return true;
+        }
+
```

Comment:
Why do we need to check both script type and language? Also wondering if it might be less confusing for this method to be implemented as
```csharp
return functions.Select(f => f.Language).Distinct().Count() <= 1
```

Reply:
Also, a better name might be "IsSingleLanguage"